dspace/solr/search/conf/solrconfig.xml

```diff
@@ -1048,9 +1048,9 @@
       <str name="uprefix">ignored_</str>
 
       <!-- capture link hrefs but ignore div attributes -->
```

Comment:
Yes, it is ignoring both hrefs and div now... BTW that seems a default comment from the configuration file, probably should be deleted altogether.